uncrustify-riot.cfg

```diff
@@ -74,3 +74,9 @@ align_on_tabstop       = TRUE      # align on tabstops
 align_enum_equ_span    = 4         # '=' in enum definition
 align_struct_init_span = 0         # align stuff in a structure init '= { }'
 align_right_cmt_span   = 3         #
+
+#
+# Special cases
+#
+
+set PROTO_WRAP ISR   # Wrap AVR ISR macros like a functions
```

Comment:
either remove `a` or the `s` at the end of function. I would do the former.

Reply:
also git grep shows that this macro is also used in other CPUs (msp430, nrf5xx, ) so the comment should not mention  AVR only.
Do you think the special case could be extended using some kind of regexp like `set PROTO_WRAP *ISR` ? This way it will also cover `UART_X_ISR`, `TIMER_X_ISR` used in other cpus.

Reply:
The wrapping problem does not occur with the other isr as they are redefinition of functions and are then used as functions. So uncrustify recognizes them as functions.

they look like follows.
```
void I2C_2_ISR(void)
void TIMER_1_ISR(void)
void UART_0_ISR_RX(void)
```